backend/worker/quivr_worker/celery_worker.py

```diff
@@ -91,18 +107,50 @@ def process_file_task(
     source_link: str | None = None,
     delete_file: bool = False,
 ):
+    if async_engine is None:
+        init_worker()
+
     logger.info(
         f"Task process_file started for file_name={file_name}, knowledge_id={knowledge_id}, brain_id={brain_id}, notification_id={notification_id}"
     )
 
-    brain_vector_service = BrainVectorService(brain_id)
     loop = asyncio.get_event_loop()
     loop.run_until_complete(
-        process_uploaded_file(
+        aprocess_file_task(
+            file_name=file_name,
+            file_original_name=file_original_name,
+            brain_id=brain_id,
+            notification_id=notification_id,
+            knowledge_id=knowledge_id,
+            source=source,
+            source_link=source_link,
+            delete_file=delete_file,
+        )
+    )
+
+
+async def aprocess_file_task(
```

Comment:
To respect project structure we should move this function in `quivr_worker/process/porcess_file.py`